lib/CleantalkSP/SpbctWP/VulnerabilityAlarm/VulnerabilityAlarmView.php

```diff
@@ -53,10 +53,40 @@ public static function showPluginAlarm($plugin_file, $plugin_report)
         return '<tr><td colspan="4">' . $vulnerability_text . '</td></tr>';
     }
 
-    public static function showThemeAlarm()
+    public static function showThemeAlarm($theme_report)
     {
-        $alarm_text = esc_html__('The theme contains known vulnerability', 'security-malware-firewall');
-        return $alarm_text;
+        $slug = $theme_report->slug;
+        $id = $theme_report->id;
+        $alarm_text = esc_html__('This theme version contains a known vulnerability', 'security-malware-firewall');
+        $alarm_cve_url = $theme_report->CVE;
+        $delete_text = esc_html__('It is strongly recommended to delete the theme.', 'security-malware-firewall');
+        $update_text = esc_html__('Updating the plugin to a version higher than ', 'security-malware-firewall');
```

Comment:
Need to be changed _Updating the **plugin** to a version higher than_ to _Updating the **theme** to a version higher than_